events/events_network.c

```diff
@@ -86,6 +95,10 @@ init(void)
 	fds = NULL;
 	fds_alloc = nfds = fdscanpos = 0;
 
+	/* Clean up the socket list at exit. */
+	if (atexit(events_network_atexit))
```

Comment:
I just realized that we could make this `atexit(events_network_shutdown)` instead of having the 1-line function `events_network_atexit`.  Would that be preferable?  I think my original thought was that `_atexit` functions would obviously be run via `atexit`, but in retrospect `_shutdown` has almost the same implication.